packages/plugin-meta-extractor/src/utils.ts

```diff
@@ -39,6 +39,48 @@ export function getExtensionConfigs(functionName: string, ast: ts.Node, checker:
   return extensionConfigs;
 }
 
+function createExtensionMatcher(functionName: string): (node: ts.Node) => boolean {
```

Comment:
Nit: as this seems more like a general function, maybe we could call it something like `createNodeMatcher` or `createFuncitonMatcher`?